backend/src/main/java/codesquard/app/user_reaction/controller/UserReactionController.java

```diff
@@ -0,0 +1,42 @@
+package codesquard.app.user_reaction.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import codesquard.app.api.response.ApiResponse;
+import codesquard.app.api.response.ResponseMessage;
+import codesquard.app.user_reaction.dto.response.UserReactionSaveResponse;
+import codesquard.app.user_reaction.service.UserReactionService;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@RequestMapping(path = "/api/reactions")
+@RestController
+public class UserReactionController {
+
+	private final UserReactionService userReactionService;
+
+	@ResponseStatus(HttpStatus.CREATED)
+	@PostMapping("/{reactionId}/issues/{issueId}")
+	public ApiResponse<UserReactionSaveResponse> saveIssueReaction(@PathVariable Long reactionId,
+		@PathVariable Long issueId) {
+		Long userId = 1L;
```

Comment:
```suggestion
	public ApiResponse<UserReactionSaveResponse> saveIssueReaction(@PathVariable Long reactionId,
		@PathVariable Long issueId, @Login AuthenticateUser user) {
		Long userId = user.getId();
```
"@Login" 애노테이션에 대한  argument resolver를 구현하였기 때문에 매개변수에 AuthenticateUser를 넣어 사용자 정보를 가져올수 있습니다.

Reply:
오 이부분은 전체적으로 변경해야 할 것 같아서 이 pr이 merge된 이후에 다른 브랜치에서 전체적으로 적용하겠습니당~ 알려주셔서 감사합니다!!